docs/walkthrough.md

```diff
@@ -414,37 +398,37 @@ see a new one:
 
 ```console
 $ kubectl get secrets -n test-ns
-NAME                  TYPE                                  DATA   AGE
-default-token-v24x9   kubernetes.io/service-account-token   3      1m
-ups-binding           Opaque                                2      37s
+NAME                    TYPE                                  DATA   AGE
+default-token-n2j75     kubernetes.io/service-account-token   3      10m
+mini-binding            Opaque                                8      91s
 ```
 
-Notice that a new `Secret` named `ups-binding` has been created.
+Notice that a new `Secret` named `mini-binding` has been created.
```

Comment:
could you add a blurb here like "At this point,we could use this secret to connect the running MariaDB database to our application running on Kubernetes."